web-common/src/features/scheduled-reports/BaseScheduledReportForm.svelte

```diff
@@ -108,6 +109,16 @@
     optional
     placeholder="1000"
   />
+  <Checkbox
+    bind:checked={$data["exportIncludeHeader"]}
+    id="exportIncludeHeader"
+    onCheckedChange={(checked) => {
+      $data["exportIncludeHeader"] = checked;
+    }}
+    inverse
+    disabled={$data["exportFormat"] === V1ExportFormat.EXPORT_FORMAT_PARQUET}
+    label="Include metadata"
+  />
```

Comment:
Should this be gated by the feature flag too? "Include metadata" doesn't feel like a sufficiently clear explanation. A tooltip could help.